server.js

```diff
@@ -2989,6 +2989,32 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+camp.route(/^\/sourcegraph\/rrc\/([\s\S]+)\.(svg|png|gif|jpg|json)$/,
+cache(function (data, match, sendBadge, request) {
+  var repo = match[1];
+  var format = match[2];
+  var apiUrl = "https://sourcegraph.com/.api/repos/" + repo + "/-/shield";
+  var badgeData = getBadgeData('dependents', data);
+  request(apiUrl, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      badgeData.text[0] = "Used by";
```

Comment:
Think this line still needs to be removed and change [Line 2997](https://github.com/badges/shields/pull/828/files#diff-78c12f5adc1848d13b1c6f07055d996eR2997) to ` var badgeData = getBadgeData('used by', data);`